incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -0,0 +1,39 @@
+package test
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Issues", func() {
+
+	const (
+		nsParent = "issues-parent"
+		nsChild = "issues-child"
+	)
+
+	BeforeEach(func() {
+		cleanupNamespaces(nsParent, nsChild)
+	})
+
+	AfterEach(func() {
+		cleanupNamespaces(nsParent, nsChild)
+	}) 
+
+	It("Should test issue #328", func() {
```

Comment:
Probably better to change the it block here to say "it should remove obsolete conditions - see issue #328"?

Reply:
done